tests/test_addons_toupdate.py

```diff
@@ -0,0 +1,610 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# License GPL-3.0 or later (http://www.gnu.org/licenses/gpl.html).
+
+from subprocess import check_call
+
+from click.testing import CliRunner
+
+from acsoo.main import main
+from acsoo.tools import working_directory
+
+
+def _checkout_to(tmpdir, git_ref):
+    check_call(['git', 'checkout', git_ref, '--quiet'], cwd=tmpdir)
+
+
+def test_addon_toupdate_00(initdir):
+    """
+    Data:
+        - addon_1 exists
+        - no requirements file
+    Test case:
+        - compare against the same revision
+        - compare requirements: YES
+    Expected result:
+        - exit code != 0 (no requirements file)
+    """
+    runner = CliRunner()
+    _checkout_to(initdir, 'cfdb4d310cd04ce0ea8098586007f7d561a1c0aa')
+    with working_directory(initdir):
+        res = runner.invoke(main, [
+            'addons',
+            'toupdate',
+            'cfdb4d310cd04ce0ea8098586007f7d561a1c0aa',
+            '--requirement',
+            './requirements.txt',
```

Comment:
`--requirement requirements.txt` is the default, so no need to repeat it in each test